pandapipes/component_models/abstract_models/branch_w_internals_models.py

```diff
@@ -6,7 +6,7 @@
 
 from pandapipes.component_models.abstract_models.branch_models import BranchComponent
 from pandapipes.component_models.component_toolbox import set_entry_check_repeat
-from pandapipes.idx_branch import ACTIVE, FROM_NODE, TO_NODE, TINIT, RHO, ETA, CP, ELEMENT_IDX
+from pandapipes.idx_branch import ACTIVE, FROM_NODE, TO_NODE, TINIT_IN, RHO, ETA, CP, ELEMENT_IDX
```

Comment:
What is this renaming of TINIT meant for? We initialize this variable as (TINIT[from_nodes] + TINIT[to_nodes]) / 2. That would mean to me that it is not the temperature at the inlet.

Reply:
I changed the name of T_OUT into TOUTINIT to be consistent. TININIT is removed.